sql/updates/world/master/2025_mm_dd_xx_world.sql

```diff
@@ -0,0 +1,7 @@
+DELETE FROM `spell_proc` WHERE `SpellId` IN (388004);
+INSERT INTO `spell_proc` (`SpellId`,`SchoolMask`,`SpellFamilyName`,`SpellFamilyMask0`,`SpellFamilyMask1`,`SpellFamilyMask2`,`SpellFamilyMask3`,`ProcFlags`,`ProcFlags2`,`SpellTypeMask`,`SpellPhaseMask`,`HitMask`,`AttributesMask`,`DisableEffectsMask`,`ProcsPerMinute`,`Chance`,`Cooldown`,`Charges`) VALUES
+(388004,0x00,4,0x00000000,0x00000000,0x00000000,0x00000400,0x0,0x0,0x0,0x4,0x0,0x0,0x0,0,0,0,0); -- Slaughtering Strikes
```

Comment:
shouldn't ProcFlag be PROC_FLAG_DEAL_MELEE_ABILITY 0x10 and SpellPhaseMask set to PROC_SPELL_PHASE_HIT 0x2?

0x4 happens after damage and visuals are done, whereas 2 happens when it hits